apps/web/package.json

```diff
@@ -44,6 +45,7 @@
     "react": "19.0.0-rc-f994737d14-20240522",
     "react-dom": "19.0.0-rc-f994737d14-20240522",
     "socket.io-react-hook": "^2.4.5",
+    "recharts": "2.13.0-alpha.5",
```

Comment:
recharts at the end?

Reply:
Is used under the hood by shad/cn charts